app/frontend/src/features/profile/actions.test.ts

```diff
@@ -0,0 +1,179 @@
+import { Empty } from "google-protobuf/google/protobuf/empty_pb";
+import type { RootState } from "../../reducers";
+import { service } from "../../service";
+import { addDefaultUser } from "../../test/utils";
+import { updateHostingPreference, updateUserProfile } from "./actions";
+
+const getUserMock = service.user.getUser as jest.Mock;
+const updateProfileMock = service.user.updateProfile as jest.Mock;
+const updateHostingPreferenceMock = service.user
+  .updateHostingPreference as jest.Mock;
+
+describe("updateUserProfile thunk", () => {
+  test("updates the store with the latest user profile info", async () => {
```

Comment:
What about using `it` for readability? Together with the first argument it then forms a full sentence.

Reply:
I used `test` as that was the advertised API, but I see they also have `it` as an alias. Yeah let's do it, more readable and less typing!